Resources/public/js/KilikTable.js

```diff
@@ -33,37 +34,42 @@ function KilikTable(id, path, options) {
             localStorage.removeItem('localStorage');
         } catch (e) {
             Storage.prototype._setItem = Storage.prototype.setItem;
-            Storage.prototype.setItem = function() {};
+            Storage.prototype.setItem = function () {
+            };
         }
     }
 
-    // for each option
-    for (optionKey in options) {
-        switch (optionKey) {
-            case "sortColumnClassSortable":
-                this.sortColumnClassSortable = options[optionKey];
-                break;
-            case "sortColumnClassSorted":
-                this.sortColumnClassSorted = options[optionKey];
-                break;
-            case "sortColumnClassSortedReverse":
-                this.sortColumnClassSortedReverse = options[optionKey];
-                break;
-            case "askForReloadDelay":
-                this.askForReloadDelay = options[optionKey];
-                break;
-            case "rowsPerPage":
-                this.rowsPerPage = options[optionKey];
-                break;
-            case "defaultHiddenColumns":
-                this.defaultHiddenColumns = options[optionKey];
-                break;
-            case "skipLoadFromLocalStorage":
-                this.skipLoadFromLocalStorage = options[optionKey];
-                break;
+    this.applyOptions = function (options) {
+        // for each option
+        for (optionKey in options) {
+            switch (optionKey) {
+                case "sortColumnClassSortable":
+                    this.sortColumnClassSortable = options[optionKey];
```

Comment:
Je pense que l'on doit pouvoir écrire  `this[optionKey] = options[optionKey]`

Ce qui donnerait :

```js
var allowedOptions = [
  "sortColumnClassSortable",
  "sortColumnClassSorted",
  "sortColumnClassSortedReverse",
  "askForReloadDelay",
  "rowsPerPage",
  "defaultHiddenColumns",
  "skipLoadFromLocalStorage"
]
for (optionKey in options) {
  if(allowedOptions.indexOf(optionKey) != -1) {  
     this[optionKey] = options[optionKey];
  }
}
```

Reply:
done